test/e2e/tests/settings/change-language.spec.ts

```diff
@@ -30,74 +28,50 @@ const selectors = {
   headerText: { text: 'الإعدادات', tag: 'h3' },
 };
 
-async function changeLanguage({ driver, languageIndex }) {
+async function changeLanguage(driver: Driver , languageIndex: number) {
   await driver.clickElement(selectors.accountOptionsMenuButton);
   await driver.clickElement(selectors.settingsOption);
 
   const dropdownElement = await driver.findElement(selectors.localeSelect);
   await dropdownElement.click();
 
-  const options = await dropdownElement.findElements(By.css('option'));
+  const options = await dropdownElement.findElements({ css: 'option' });
   await options[languageIndex].click();
 }
 
-describe('Settings - general tab, validate the change language functionality:', function () {
-  it('User selects "Español" language and verify that changing the language from the default', async function () {
-    const languageIndex = 10;
+describe('Settings - general tab @no-mmi', function (this: Suite) {
+  it('validate the change language functionality', async function () {
+    let languageIndex = 10;
+
     await withFixtures(
       {
         fixtures: new FixtureBuilder().build(),
         ganacheOptions: defaultGanacheOptions,
-        title: this.test.fullTitle(),
+        title: this.test?.fullTitle(),
       },
 
-      async ({ driver }) => {
+      async ({ driver }: { driver: Driver }) => {
         await unlockWallet(driver);
+        await changeLanguage(driver, languageIndex);
 
-        await changeLanguage({ driver, languageIndex });
-
-        // Validate the label changes
+        // Validate the label changes to spanish
         const isLanguageLabelChanged = await driver.isElementPresent(
           selectors.labelSpanish,
         );
         assert.equal(isLanguageLabelChanged, true, 'Language did not change');
-      },
-    );
-  });
-
-  it('User selects "Español" language from the dropdown and change back to english and verify that the word is correctly changed back to english', async function () {
-    let languageIndex = 10;
-    await withFixtures(
-      {
-        fixtures: new FixtureBuilder().build(),
-        ganacheOptions: defaultGanacheOptions,
-        title: this.test.fullTitle(),
-      },
 
-      async ({ driver }) => {
-        await unlockWallet(driver);
-        await changeLanguage({ driver, languageIndex });
-        const isLanguageLabelChanged = await driver.isElementPresent(
-          selectors.primaryCurrencyLabel,
-        );
-        assert.equal(isLanguageLabelChanged, true, 'Language did not change');
-
-        await driver.assertElementNotPresent('.loading-overlay__spinner', {
-          waitAtLeastGuard: 100,
-        });
+        await driver.refresh();
 
+        // Change back to english and verify that the word is correctly changed back to english
```

Comment:
Language names like `English` and `Spanish` should always be capitalized

Reply:
Ok done.